### ISSUE TITLE
undefined is not a valid semantic version

### ISSUE THREAD

**Issue by sembrestels** [aragon/aragon-cli#1124]
## 🐛 Bug Report

Error on starting the counter app.

### Have you read the [Contributing Guidelines on issues](https://github.com/aragon/aragon-cli/blob/master/CONTRIBUTING.md#ways-to-contribute)?

Yes

#### Mainnet or testnet?

RPC

#### Environment

- OS and OS version: Ubuntu 19.04
- CLI version: 7.0.2

## To Reproduce

```
git clone https://github.com/aragon/aragon-react-boilerplate
cd aragon-react-boilerplate
npm install
nano node_modules/.bin/aragon # remove the --no-depracation flag to make it work (see #1123)
npm start
```

It also happens with `npx create-aragon-app <app-name> react`.

## Expected behavior

It should start the Counter app.

## Actual Behavior
```
  ✔ Check port
  ✔ Start a local Ethereum network
  ✔ Setup before publish
  ✔ Prepare for publish
  ❯ Publish app to aragonPM
    ✖ Publish placeholder-app-name.open.aragonpm.eth
      → undefined is not a valid semantic version
    Fetch published repo
    Create Organization
    Start Client
✖ undefined is not a valid semantic version
```

**Comment by ajsantander** [comment]
Thanks for reporting this @sembrestels !!

**Comment by sembrestels** [comment]
That same issue raises again in aragon-cli v7.1.1.

**Comment by 0xGabi** [comment]
Fixed by #1510
